UPGRADE.md

```diff
@@ -0,0 +1,6 @@
+# Upgrade
+
+## From 1.x to 2.x
+
+- Replace `HealthEngine\I18n` references with `Healthengine\I18n`.
+- Be prepared for invalid configuration/translation files to start throwing exceptions.
```

Comment:
Is there a "clear cache" instruction that would help solve this?

Reply:
Cache does not help with this problem